### PR TITLE
Story/CCP-225

### PR DESCRIPTION
**Pull request by mkarthik90**

### CODE REVIEW

**Comment by mkarthik90** [comment]
Need to include the changes for creating wrappers from concept list screen.

File: Conceptpower+Spring/src/main/webapp/WEB-INF/views/home.jsp [hunk 1]
```diff
@@ -141,10 +142,29 @@ function prepareMergeConcept(conceptId) {
...
+      },
+      type : "GET",
+      url : "${pageContext.servletContext.contextPath}/rest/Concept",
+      data : {
+        id : conceptId
+      },
+      success : function(details) {
+        $.each(details.conceptEntries, function (index, conceptEntry) {
+          window.location = '${pageContext.servletContext.contextPath}/auth/conceptlist/addconceptwrapper?word=' + conceptEntry.lemma + '&selectedConceptList=' + conceptEntry.conceptList + '&description=' + conceptEntry.description + '&selectedType=' + conceptEntry.type + '&wrapperids=' + conceptEntry.id;
+        });
```
Comment (jdamerow) [inline Conceptpower+Spring/src/main/webapp/WEB-INF/views/home.jsp]: I'm pretty sure you have all this info when you build the results page, why do you make an extra concept request?

**Comment by mkarthik90** [comment]
@jdamerow  Addressed review comments. Removed ajax call and replaced with the values from concept entry. Passed the individual values instead of concept entry object to the method. For some reason js is not able to process the concept entry object so passed the individual values to createWrapper() method.

**Comment by jdamerow** [comment]
ok to test

**Pull request merged**